R/stats-lm-tidiers.R

```diff
@@ -279,5 +279,52 @@ process_lm <- function(ret, x, conf.int = FALSE, conf.level = .95,
 }
 
 
+#' TODO: documentation
+#' @evalRd return_tidy(regression = TRUE)
+#' 
+#' @examples 
+#' CarReg = lm(formula = speed ~., data = cars)
+#' 
+#' # Standardize coefficients
+#' library(lm.beta)
+#' CarRegS = lm.beta::lm.beta(CarReg)
+#' 
+#' # Table format
+#' tidy(CarRegS)
+#'
+#' @export
+#' @family lm tidiers
+tidy.lm.beta <- function (x, conf.int = FALSE, conf.level = 0.95, exponentiate = FALSE, 
+                          quick = FALSE, ...) {
+  if (quick) {
```

Comment:
This looks like a copy-paste of `tidy.lm()`, which is a very reasonable place to start. However, `tidy.lm()` is a mess and is way more complicated than it needs to be. Would you mind attempting something more self-contained in the vein of `tidy.nls()` for example (https://github.com/tidymodels/broom/blob/master/R/stats-nls-tidiers.R#L32)?